src/phoenix/utilities/logging.py

```diff
@@ -1,10 +1,18 @@
 # A collection of printing and logging utilities
 
-from typing import Any
+from typing import Any, List
 
 from tqdm.auto import tqdm
 
 
 def printif(condition: bool, *args: Any, **kwargs: Any) -> None:
     if condition:
         tqdm.write(*args, **kwargs)
+
+
+def log_a_list(list_of_str: List[str], join_word: str) -> str:
+    if list_of_str is None or len(list_of_str) == 0:
+        return ""
+    if len(list_of_str) == 1:
+        return list_of_str[0]
+    return f"{', '.join(map(str, list_of_str[:-1]))} {join_word} {list_of_str[-1]}"
```

Comment:
Not sure I see the need for this since `"and, ".join(list)` is pretty common. Also fine to just log a list directly since it gets formatted nicely.

Reply:
Completely up to you, I find it nicer to not have "and" everywhere. IMO it's confusing, and it's a linguistic device (I believe in English may be called Polysyndeton) that IMO should be avoided unless intended. But I'm a stickler for words haha.

Reply:
It's a pet peeve of mine specially in Spanish, but like I said, can remove no problem